app/controllers/posts_controller.rb

```diff
@@ -6,5 +6,24 @@ def index
 
   def show
     @post = Post.find(params[:id])
+    @author = User.find(params[:user_id])
+    @comments = Comment.where(post_id: params[:id])
+    @comment = @post.comments.new
+  end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(params.require(:post).permit(:title, :text))
+    @post.user = current_user
+    if @post.save
+      flash[:success] = 'Post created successfully'
+      redirect_to user_post_path(@post.user, @post)
+    else
+      flash.now[:error] = 'Something went wrong'
+      render :new, status: 422
+    end
```

Comment:
- [ ] Great job making the controller for the posts and comments to create a new post and a new comment, However we need also if the user press on a like it should be there so we need to add a `create` method to create the like as well so kindly add a button as well to add the like 💪🏻🙏🏻✨🔥


![image](https://github.com/julzedz/tech-krunch/assets/58553711/05886271-3e1a-4f8c-bb56-68079057b0f6)